### PR TITLE
Release Candidate

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**
This PR upgrades:

1. ManagementService-v3
2. ManagmentNotificationService-v3
3. TransferService-v4
4. BalancePlatformTransferNotification-v4

### CODE REVIEW

**Review by michaelpaul — APPROVED** [review]

**Review by michaelpaul — CHANGES_REQUESTED** [review]
WIP

**Review by Aleffio — COMMENTED** [review]
minor, let's also update the version of the hateoas links in the mocks
https://github.com/Adyen/adyen-dotnet-api-library/blob/1b6242d50f4921262cbfa66f26fcefd79067b21d/Adyen.Test/mocks/management/list-merchant-accounts.json#L4

**Comment by AlexandrosMor** [comment]
@Aleffio check again ?

**Review by Aleffio — DISMISSED** [review]

**Review by michaelpaul — APPROVED** [review]

**Review by michaelpaul — CHANGES_REQUESTED** [review]
WIP

File: Makefile [hunk 1]
```diff
@@ -22,13 +22,16 @@ PosTerminalManagement: spec=TfmAPIService-v1
...
-Management: spec=ManagementService-v1
+Management: spec=ManagementService-v3
 LegalEntityManagement: spec=LegalEntityService-v3
 PlatformsAccount: spec=AccountService-v6
 PlatformsFund: spec=FundService-v6
 PlatformsNotificationConfiguration: spec=NotificationConfigurationService-v6
 PlatformsHostedOnboardingPage: spec=HopService-v6
-Transfers: spec=TransferService-v3
+Transfers: spec=TransferService-v4
+ManagmentNotificationService: spec=ManagementNotificationService-v3
```
Comment (Aleffio) [inline Makefile:32]:
```suggestion
ManagementNotificationService: spec=ManagementNotificationService-v3
```

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ openapi-generator-cli:=java -jar $(openapi-generator-jar)
 
 
 generator:=csharp-netcore
-services:=BalanceControl BalancePlatform BinLookup Checkout DataProtection LegalEntityManagement Management Payment Payout PosTerminalManagement Recurring StoredValue Transfers
+services:=BalanceControl BalancePlatform BinLookup Checkout DataProtection LegalEntityManagement Management Payment Payout PosTerminalManagement Recurring StoredValue Transfers ManagmentNotificationService BalancePlatformTransferNotification
```
Comment (Aleffio) [inline Makefile:8]:
```suggestion
services:=BalanceControl BalancePlatform BinLookup Checkout DataProtection LegalEntityManagement Management Payment Payout PosTerminalManagement Recurring StoredValue Transfers ManagementNotificationService BalancePlatformTransferNotification
```

**Pull request closed without merging**